src/components/VAudioTrack/layouts/VFullLayout.vue

```diff
@@ -58,25 +58,45 @@
         </div>
       </div>
 
-      <VDownloadButton
-        class="ms-auto order-1 lg:order-2"
-        :formats="getFormats(audio)"
+      <VButton
+        as="VLink"
+        :href="audio.foreign_landing_url"
         :size="isSmall ? 'small' : 'medium'"
-      />
+        class="ms-auto order-1 lg:order-2 font-bold"
+      >
+        {{ $t('download-button.download') }}
+      </VButton>
     </div>
   </div>
 </template>
 
 <script>
 import { computed, defineComponent } from '@nuxtjs/composition-api'
 
-import VDownloadButton from '~/components/VDownloadButton.vue'
+import VButton from '~/components/VButton.vue'
 import VLink from '~/components/VLink.vue'
 
 export default defineComponent({
   name: 'VFullLayout',
-  components: { VDownloadButton, VLink },
-  props: ['audio', 'size', 'status', 'currentTime'],
+  components: { VButton, VLink },
+  props: {
+    audio: {
+      type: Object,
+      required: true,
+    },
+    size: {
+      type: String,
+      validation: (v) => ['s', 'm', 'l'].includes(v),
+    },
+    status: {
+      type: String,
+      validation: (v) => ['playing', 'played', 'ejected'].includes(v),
```

Comment:
I think this should be "paused" instead of "ejected". 
```suggestion
      validation: (v) => ['playing', 'played', 'paused'].includes(v),
```